### PR TITLE
Update config.ini.example

### PR DESCRIPTION
**Pull request by Calvinthesneak**
Went through the command line wiki page and added all the --flag options.  Tried to classify them by sections.  Might be utterly useless and too messy.....

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Calvinthesneak** [comment]
Tried to update the list of config commands available....

File: config/config.ini.example [hunk 1]
```diff
@@ -1,61 +1,118 @@
...
+
+
+# Captcha Settings(See https://rocketmap.readthedocs.io/en/develop/extras/captchas.html)
+#captcha-solving:		# Enables captcha solving.(default false)
+#captcha-key:			# 2Captcha API Key
+#captcha-dsk:			# Pokemon Go captcha data-sitekey
+#manual-captcha-domain:	# Domain to where captcha tokens will be sent (default: http://127.0.0.1:5000)
+#manual-captcha-refresh:# Time available before captcha page refreshes (default 30 seconds)
+#manual-captcha-timeout:# Maximum time captchas will wait for manual captcha solving. On timeout, if enabled, 2Captcha will be used to solve captcha. (default 0 - disabled)
 #status-name:           # enables writing status updates to the database - if you use multiple processes, each needs a unique value
```
Comment (k4n30) [inline config/config.ini.example]: You have this twice (this one should go)

File: config/config.ini.example [hunk 2]
```diff
@@ -1,61 +1,118 @@
...
+#wh-threads:			# Number of webhook threads; increase if the webhook queue falls behind.
 #wh-retries:            # Number of times to retry sending webhook data on failure (default: 5).
 #wh-timeout:            # Timeout (in seconds) for webhook requests (default: 2).
 #wh-backoff-factor:     # Factor (in seconds) by which the delay until next retry will increase (default: 0.25).
 #wh-lfu-size:           # Webhook LFU cache max size (default: 1000).
+
+# Status Page
 #print-status:          # Enable ENTER to switch between log view and a status view.  Use 'status' to start in status view and 'logs' to start in log view
+#stats-log-timer:		# In log view, list per hr stats every X seconds.
+#status-name:			# Enable status page database update using STATUS_NAME as main worker name.
```
Comment (k4n30) [inline config/config.ini.example]: You have this twice (this one should stay)

**Comment by k4n30** [comment]
Also, you have missed the altitude variables - https://github.com/RocketMap/RocketMap/pull/1645

And the last line https://github.com/RocketMap/RocketMap/pull/1868/files#diff-370690a23e7c2bcd1b41d6aa181f8d42R119 needs to be updated to reflect that you can use an account file so you don't have to have a user & password

Thanks for doing this btw, I have been wanting to do this for awhile

**Comment by Obihoernchen** [comment]
This is a duplicate of #1844

**Comment by Arengo** [comment]
Yup, unnecessary work :-/ #1844

**Comment by Calvinthesneak** [comment]
Well damn, I even looked through proposed stuff when I started.  Guess I must be losing it.  I started with the config wiki page, but realized there was little point if the example wasn't up to date.  At any rate glad to see someone did it.  I'll continue working on the wiki page tonight.

**Pull request closed without merging**